### PR TITLE
Document preview API routes

### PR DESCRIPTION
**Pull request by fpagnoux**
Connected to openfisca/openfisca-web-api#129

### CODE REVIEW

**Comment by fpagnoux** [comment]
Preview visible [here](http://openfisca.readthedocs.io/en/add-new-api-doc/)

File: doc/source/parameter.rst [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+=============
+
+.. http:get:: /parameter/<parameter_id>
+
+    Get data about the ``<parameter_id>`` parameter
+
+    :>json description: Parameter description.
+    :>json id: Parameter id.
+    :>json values: Parameter values history. Only present if the parameter is a simple one. A ``null`` value means that the parameter does not exist anymore.
+    :>json brackets: Marginal scale history. Only present if the parameter is a marginal scale. A ``null`` value means that the scale does not exist anymore.
```
Comment (MattiSG) [inline doc/source/parameter.rst]: > A ``null`` value means that the parameter does not exist anymore.

This is misleading: `values` can never be null.

Also, I see nowhere documented the types of these properties: `description` and `id` are `String`, while `values` and `brackets` are `Object`…

File: doc/source/parameter.rst [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+    :resheader Country-Package: The name of the country package served by the API.
+    :resheader Country-Package-Version: The version of the country package served by the API.
+    :statuscode 200: no error.
```
Comment (MattiSG) [inline doc/source/parameter.rst]: “The requested parameter description is sent back in the response body.”

File: doc/source/parameters.rst [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+``/parameters``
+==============
+
+.. http:get:: /parameters
+
+    All the parameters contained in the tax and benefit system
+
+    :resheader Country-Package: The name of the country package served by the API
+    :resheader Country-Package-Version: The version of the country package served by the API
+    :statuscode 200: no error
```
Comment (MattiSG) [inline doc/source/parameters.rst]: “The list of all parameters is sent back in the response body.”

File: doc/source/parameters.rst [hunk 4]
```diff
@@ -0,0 +1,44 @@
+==============
+``/parameters``
+==============
+
+.. http:get:: /parameters
+
+    All the parameters contained in the tax and benefit system
```
Comment (MattiSG) [inline doc/source/parameters.rst]: Ok, but I could mostly infer this with the name. What I'm looking for in the doc is the response format. I need to know that I will get an object whose keys will be parameters IDs and values objects with a single property, “description”, which is a String, and that I can use the ID to navigate to more information with the `parameter` endpoint.

File: doc/source/parameter.rst [hunk 5]
```diff
@@ -0,0 +1,90 @@
+=============
+``/parameter``
+=============
+
+.. http:get:: /parameter/<parameter_id>
+
+    Get data about the ``<parameter_id>`` parameter
```
Comment (MattiSG) [inline doc/source/parameter.rst]: “data about” could be inferred with the name of the endpoint. “all information contained in the loaded tax and benefit system about” at least tells me that I will get a full dump. Maybe “the type, description and values”?

File: doc/source/variable.rst [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+=============
+
+.. http:get:: /variable/<variable_id>
+
+    Get data about the ``<variable_id>`` variable
+
+    :>json defaultValue: Variable default value.
+    :>json definitionPeriod: Definition period of the variable. Possible values : ``YEAR``, ``MONTH``, ``ETERNITY``.
+    :>json description: Variable description.
+    :>json entity: Entity for which the variable is defined. For instance, ``person``, ``household``…
```
Comment (MattiSG) [inline doc/source/variable.rst]: Link to the reference manual.

File: doc/source/variable.rst [hunk 7]
```diff
@@ -0,0 +1,59 @@
...
+``/variable``
+=============
+
+.. http:get:: /variable/<variable_id>
+
+    Get data about the ``<variable_id>`` variable
+
+    :>json defaultValue: Variable default value.
+    :>json definitionPeriod: Definition period of the variable. Possible values : ``YEAR``, ``MONTH``, ``ETERNITY``.
+    :>json description: Variable description.
```
Comment (MattiSG) [inline doc/source/variable.rst]: All properties need types (see comments on `parameter`).

**Pull request merged**